OCI ConveyorPacker test failures

Not seen in CI, as skipped when run with -short

```
$ sudo go test -v -count 1 -run TestOCI ./internal/pkg/build/sources/...
=== RUN   TestOCIConveyorDocker
    conveyorPacker_oci_test.go:76: failed to Get from docker://alpine: no child with platform  in index alpine
--- FAIL: TestOCIConveyorDocker (1.00s)
=== RUN   TestOCIConveyorDockerArchive
--- PASS: TestOCIConveyorDockerArchive (1.87s)
=== RUN   TestOCIConveyorDockerDaemon
--- PASS: TestOCIConveyorDockerDaemon (1.82s)
=== RUN   TestOCIConveyorOCIArchive
--- PASS: TestOCIConveyorOCIArchive (0.75s)
=== RUN   TestOCIConveyorOCILayout
--- PASS: TestOCIConveyorOCILayout (0.46s)
=== RUN   TestOCIPacker
    conveyorPacker_oci_test.go:272: failed to Get from docker://alpine: no child with platform  in index alpine
--- FAIL: TestOCIPacker (0.90s)
FAIL
FAIL	github.com/sylabs/singularity/v4/internal/pkg/build/sources	6.816s
FAIL
```

--- a/internal/pkg/build/sources/conveyorPacker_oci_test.go
+++ b/internal/pkg/build/sources/conveyorPacker_oci_test.go
@@ -17,6 +17,7 @@ import (
 
 	"github.com/sylabs/singularity/v4/internal/pkg/build/sources"
 	"github.com/sylabs/singularity/v4/internal/pkg/cache"
+	"github.com/sylabs/singularity/v4/internal/pkg/ociplatform"
 	testCache "github.com/sylabs/singularity/v4/internal/pkg/test/tool/cache"
 	"github.com/sylabs/singularity/v4/pkg/build/types"
 	useragent "github.com/sylabs/singularity/v4/pkg/util/user-agent"
@@ -66,7 +67,11 @@ func TestOCIConveyorDocker(t *testing.T) {
 	}
 
 	b.Opts.ImgCache = imgCache
-
+	p, err := ociplatform.DefaultPlatform()
+	if err != nil {
+		t.Fatalf("failed to get DefaultPlatform: %v", err)
+	}
+	b.Opts.Platform = *p
 	cp := &sources.OCIConveyorPacker{}
 
 	err = cp.Get(context.Background(), b)
@@ -264,6 +269,11 @@ func TestOCIPacker(t *testing.T) {
 	imgCache, cleanup := setupCache(t)
 	defer cleanup()
 	b.Opts.ImgCache = imgCache
+	p, err := ociplatform.DefaultPlatform()
+	if err != nil {
+		t.Fatalf("failed to get DefaultPlatform: %v", err)
+	}
+	b.Opts.Platform = *p
 
 	err = ocp.Get(context.Background(), b)
 	// clean up tmpfs since assembler isn't called

